### DISCUSSION TITLE
Formalising CounterExamples in Topology

### DISCUSSION THREAD

**Discussion by Shraze97** [pi-base/data#480] [General]
Hello all! I am a 5th year BS-MS student from IISc and I have been working on formalising Counter-Examples in Topology from this book :-[Counter Examples in Topology](https://en.m.wikipedia.org/wiki/Counterexamples_in_Topology) , here is my website[CounterExamples](https://shraze97.github.io/CounterE0xamples1/CounterExamples.html). So, any comments about my code will be deeply appreciated.The main objective of this project is to make Counter Examples and some tactics so that for any given conjecture in topology, one can use a formal tactic to specialize it for that particular topological space and give an instant disproof of it.

**Comment by StevenClontz** [comment]
Good week to share this - @jamesdabbs and I are hanging out at https://aimath.org/workshops/upcoming/cyberinfrastructure/ this week and working on better Lean/pi-Base integration. Will ponder this more in the near future...
